pkgs/top-level/all-packages.nix

```diff
@@ -6808,7 +6808,8 @@ with pkgs;
 
   jadx = callPackage ../tools/security/jadx { };
 
-  jamesdsp = libsForQt5.callPackage ../applications/audio/jamesdsp { };
+  jamesdsp = libsForQt5.callPackage ../applications/audio/jamesdsp { usePipewire = true; };
```

Comment:
I think we should default pipe wire to true then. This makes overrides etc easier.